### PR TITLE
update ocp-28630

### PR DESCRIPTION
**Pull request by chao007**
`Given SCC "privileged" is added to the "csi-provisioner" service account` This step will restore scc 
` if @result[:success]
    teardown_add {
      _res = nil
      wait_for(60, interval: 5) {
        _res = _admin.cli_exec(_teardown_command, **_opts)
        _res[:success]
      }
      raise "could not restore SCC of #{which} #{type}" unless _res[:success]
    }
`
So After upgrade,  service account has no privileged scc.

@qinpingli @duanwei33 Please help to review.

### CODE REVIEW

File: features/upgrade/storage/upgrade.feature [hunk 1]
```diff
@@ -191,10 +191,18 @@ Feature: Storage upgrade tests
     When I run the :apply client command with:
       | f | csi-rbac.yaml | 
     Then the step should succeed
-    Given SCC "privileged" is added to the "csi-provisioner" service account
-    Given SCC "privileged" is added to the "csi-attacher" service account
-    Given SCC "privileged" is added to the "csi-snapshotter" service account
-    Given SCC "privileged" is added to the "csi-plugin" service account
+    When I run the :oadm_policy_add_scc_to_user admin command with:
+      | scc           | privileged      |
+      |serviceaccount | csi-provisioner |
```
Comment (pruan-rht) [inline features/upgrade/storage/upgrade.feature:196]: a space between `|` and `serviceaccount` is needed... pls apply to the rest of the scenario.

**Comment by liangxia** [comment]
@chao007   Updated the steps in https://github.com/openshift/verification-tests/pull/1435 to support this one. Would you like to have a try ?

**Pull request merged**